File: 02.PYTHON/pg-166-169/pg_166_06.py
print("");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPPYPYPYPYPYPYPYPPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("READ A TWO-DIGITS INTEGER LESS THAN 20 AND DETERMINE IF IT'S A PRIME NUMBER");
print("PYPYPYPYPYPYPYPYPYPYPYPYPYPYPYPPYPYPYPYPYPYPYPPYPYPYPYPYPYPYPYPYPYPYPYPYPYP");
print("");

num = int(input("Write a two-digits integer: "));
prime = True;

if(num > 0):
    num *= -1;

if(num >= 10 and num < 20):

    for i in range(2, num):
    
        if(num % i == 0):
            prime = False;
            break;

    if(prime):
    
        print("The written number is prime.");
    
    else:
    
        print("The written number isn't prime.");
    

elif(num <= 9 or num >= 100):

    print("The written number doesn't have two digits. Please Try again!");

else:

    print("The written number isn't less than 20.");

File: 01.JAVA/pg-166-169/pg_166_06.java
import java.util.Scanner;

public class pg_166_06
{
    public static void main(String[]args)
    {
        Scanner enter = new Scanner(System.in);
        int num;
        boolean prime = true;

        System.out.println("");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("READ A TWO-DIGITS INTEGER LESS THAN 20 AND DETERMINE IF IT'S A PRIME NUMBER");
        System.out.println("JVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJVJ");
        System.out.println("");

        System.out.print("Write a two-digis integer: ");
        num = enter.nextInt();

        if(num < 0)
        {
            num *= -1;
        }

        if(num >= 10 && num < 20)
        {
            for(int i = 2; i < num; i++)
            {
                if(num % i == 0)
                    prime = false;
                    break;
            }
            if(prime)
            {
                System.out.println("The written number is prime.");
            }
            else
            {
                System.out.println("The written number isn't prime.");
            }
        }
        else if(num <= 9 || num >= 100)
        {
            System.out.println("The written number doesn't have two digits. Please try again!");
        }
        else
        {
            System.out.println("The written number isn't less than 20.");
        }
        
    }
}